com.unity.multiplayer.mlapi/Runtime/Core/SnapshotSystem.cs

```diff
@@ -336,60 +403,88 @@ public void Store(ulong networkObjectId, int behaviourIndex, int variableIndex,
             k.NetworkObjectId = networkObjectId;
             k.BehaviourIndex = (ushort)behaviourIndex;
             k.VariableIndex = (ushort)variableIndex;
+            k.TickWritten = m_NetworkManager.NetworkTickSystem.GetTick();
 
             int pos = m_Snapshot.Find(k);
             if (pos == Entry.NotFound)
             {
-                pos = m_Snapshot.AddEntry(networkObjectId, behaviourIndex, variableIndex);
+                pos = m_Snapshot.AddEntry(k);
             }
 
-            // write var into buffer, possibly adjusting entry's position and length
+            WriteVariableToSnapshot(m_Snapshot, networkVariable, pos);
+        }
+
+        private void WriteVariableToSnapshot(Snapshot snapshot, INetworkVariable networkVariable, int index)
+        {
```

Comment:
Same problem as already mentioned. Using a network write for something which should not be a network write.